library/src/main/java/com/erkutaras/showcaseview/ShowcaseManager.java

```diff
@@ -233,7 +235,10 @@ private float calculateRadius() {
             float x = view.getWidth() / 2;
             float y = view.getHeight() / 2;
             float radius = (float) Math.sqrt(Math.pow(x, 2) + Math.pow(y, 2));
-            return radius + ShowcaseUtils.convertDpToPx(INNER_MARGIN_OF_FOCUS_AREA_ON_DP);
+            Display display = ((Activity) context).getWindow().getWindowManager().getDefaultDisplay();
```

Comment:
I have added new function to support dynamic margin as ´builder.marginFocusArea(intInDp)´ in v1.2.0